mentoring/tests/integration/xml/assessment_1.xml

```diff
@@ -14,32 +14,32 @@
 
     <pb-answer name="goal" question="What is your goal?" />
 
-    <pb-mcq name="mcq_1_1" question="Do you like this MCQ?" correct_choices="yes">
+    <pb-mcq name="mcq_1_1" question="Do you like this MCQ?" correct_choices='["yes"]'>
```

Comment:
@sarina new instances would serialize like that. Existing instances would need to be converted using transformations in `xml_changes.py`. I believe @bradenmacdonald is working on creating [migration script](https://github.com/open-craft/problem-builder/pull/2) ( @bradenmacdonald please correct me if I'm wrong)